File: back/src/main/java/com/speakerspace/dto/EventDTO.java
package com.speakerspace.dto;

import lombok.Builder;

@Builder
public record EventDTO(
        String idEvent,
        String eventName,
        String description,
        String endDate,
        String url,
        String startDate,
        Boolean online,
        String location,
        Boolean privateEvent,
        String webLinkUrl,
        Boolean finished,
        String userCreateId,
        String conferenceHallUrl,
        String teamId,
        String timeZone,
        String logoBase64,
        String type
) {}

File: front/src/app/feature/admin-management/type/event/eventDTO.ts
export type EventDTO = {
  teamUrl?: string;
  idEvent?: string;
  eventName: string;
  description?: string;
  startDate?: string;
  endDate?: string;
  online?: boolean;
  url?: string;
  location?: string;
  privateEvent?: boolean;
  webLinkUrl?: string;
  finished?: boolean;
  userCreateId?: string;
  conferenceHallUrl?: string;
  teamId: string | null;
  timeZone?: string;
  logoBase64?: string | null;
  type: string;
}
